src/sections/Meshmap/Meshmap-design/Meshmap_Mobile_swiper/mobile-swiper.style.js

```diff
@@ -1,143 +1,177 @@
 import styled from "styled-components";
 
-
 const SwiperContainer = styled.div`
-
-@media (min-width: 769px) {
+  @media (min-width: 768px) {
     display: none;
-}
-
-.swiper-slide{
-  border-radius: 20px;
-}
+  }
 
-.card.light {
-  box-shadow: none;
+  .swiper-slide {
+    border-radius: 20px;
+  }
 
-  background: linear-gradient(
-    266.65deg,
-    #00b39f 40.19%,
-    #00B39F 74.31%,
-    #3c494f 92.89%
-  )
-}
+  .card.light {
+    box-shadow: none;
 
-.card.dark {
-  background: linear-gradient(
-    266.65deg,
-    #00584e 40.19%,
-    #2e5363 74.31%,
-    #212529 92.89%
-  );
-}
+    background: linear-gradient(
+      266.65deg,
+      #00b39f 40.19%,
+      #00b39f 74.31%,
+      #3c494f 92.89%
+    );
+  }
 
+  .card.dark {
+    background: linear-gradient(
+      266.65deg,
+      #00584e 40.19%,
+      #2e5363 74.31%,
+      #212529 92.89%
+    );
+  }
 
- .card{
+  .card {
     /* box-shadow: rgb(38, 57, 77, 0.5) 0px 20px 30px -10px; */
     /* box-shadow: none; */
 
-  /* background: linear-gradient(
+    /* background: linear-gradient(
     266.65deg,
     #00b39f 40.19%,
     #477e96 74.31%,
     #3c494f 92.89%
 
   ); */
 
+    height: 70vh;
+    padding: 1rem;
+    border-radius: 20px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+  .card img {
+    height: 40%;
+    width: 100%;
+    /* margin-left: 1.5rem; */
+  }
+  .card h2 {
+    text-align: center;
+    color: white;
+  }
+  .card p {
+    text-align: center;
+    color: white;
+  }
+  .card a {
+    margin-top: 1rem;
+    color: white;
+  }
+  @media (max-width: 670px) {
+    width: 68vw;
+    .card {
+      width: 64vw;
+      height: 60vh;
+    }
+    .card img {
+      height: 30%;
+      width: auto;
+    }
+  }
 
-  height:70vh;
-  padding:1rem;
-  border-radius: 20px;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-}
-.card img{
-  height:40%;
-  width:100%;
-  /* margin-left: 1.5rem; */
-}
-.card h2{
-  text-align: center;
-  color:white;
-}
-.card p{
-  text-align: center;
-  color:white;
-}
-.card a{
-  margin-top: 1rem;
-  color: white;
-}
-@media (max-width:670px){
-.card img{
-  height: auto;
-  width: auto;
-}
-}
-
-@media (max-width:577px){
-
-  h2{
+  @media (max-width: 577px) {
+    .card {
+      width: 62vw;
+    }
+    h2 {
       font-size: 1.4rem;
-
-  }
-  p{
+    }
+    p {
       font-size: 1rem;
+    }
   }
 
-}
-
-@media (max-width:520px){
-
-    h2{
-        font-size: 1.2rem;
+  @media (max-width: 520px) {
+    .card {
+      height: 55vh;
     }
-    p{
-        font-size: 0.8rem;
+    h2 {
+      font-size: 1.2rem;
+    }
+    p {
+      font-size: 0.8rem;
     }
-
   }
-  @media (max-width:470px){
-    .card{
-      /* height:50vh; */
+  @media (max-width: 490px) {
+    .card {
+      height: 55vh;
```

Comment:
Thanks @ShravaniAK, the reduced height of the cards looks good on mobile view. Why were the formatting changes pushed alongside? please revert them.